esp/src/eclwatch/LZBrowseWidget.js

```diff
@@ -800,7 +812,7 @@ define([
                     lang.mixin(item, lang.mixin({
                         targetName: item.displayName,
                         targetRecordLength: "",
-                        targetRowTag: "",
+                        targetRowTag: "tag",
```

Comment:
Agree - while it doesn't really matter, using "Row" would be a better default (as that is the default tag for an outputted xml dataset).